app/views/posts/index.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Display all posts</h1>
\ No newline at end of file
+<img src="<%= asset_path('im12.jpg') %>" alt="User Photo">
```

Comment:
- You have done great so far :+1: But there are changes and requirements you still need to meet before moving to the next milestone. :+1: 

- [ ]  I noticed there's an issue with the migration order – the migration to add foreign keys assumes the existence of the "comments" table, which is created in a later migration.

To fix this, we should ensure that the migrations run in the correct order. Here's a quick guide:

**Update Migration Timestamps:**
Make sure the migration file **20231115151208_add_foreign_keys.rb** has a timestamp earlier than the migration that creates the "comments" and other tables. If not, consider updating the timestamp to an earlier date.."  :wave: :small_airplane: 

- - Screenshot reference:
![Selection_040](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/29e0a067-74b1-499c-9cb2-38313c7198f4)